File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_18.go
package main
import ("fmt"
        
)

func main() {
    var x1 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&x1)
    var y1 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&y1)
    var x2 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&x2)
    var y2 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&y2)
    k := (y2 - y1) / (x2 - x1)
    b := y1 - (y2 - y1) / (x2 - x1) * x2
    fmt.Println (k, b)
}

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_18.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
x1 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
y1 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
x2 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
y2 = gets.to_i
k = (y2 - y1) / (x2 - x1)
b = y1 - (y2 - y1) / (x2 - x1) * x2
print ("#{k} #{b}")